osdk/src/commands/build/mod.rs

```diff
@@ -208,6 +208,13 @@ fn build_kernel_elf(
         "-C panic=unwind",
     ]);
 
+    if matches!(arch, Arch::X86_64) {
+        // This is a workaround for <https://github.com/asterinas/asterinas/issues/839>.
+        // It makes running on Intel CPUs after Ivy Bridge (2012) faster, but much slower
+        // on older CPUs.
+        rustflags.push("-Cllvm-args=-x86-use-fsrm-for-memcpy");
```

Comment:
https://github.com/asterinas/asterinas/issues/839#issuecomment-2135216652, so perhaps we need `target_feature = "ermsb"` instead (https://github.com/rust-lang/compiler-builtins/blob/c04eb9e1afb72bdf943f5e5d77b3812f40526602/src/mem/x86_64.rs#L24-L28)?

Reply:
Yep, ur right that by using `ermsb` the problems are solved, while llvm args have no effect since the compiler builtin is rust-specific.

But doing so we will have an annoying warning.

```
warning: unstable feature specified for `-Ctarget-feature`: `ermsb`
  |
  = note: this feature is not stably supported; its behavior can change in the future
```